lib/matobj2.gd

```diff
@@ -829,7 +828,7 @@ DeclareConstructor( "NewZeroVector", [ IsVectorObj, IsSemiring, IsInt ] );
 ##  <Ref Attr="BaseDomain" Label="for a vector object"/> values.
 ##  <P/>
 ##  Similarly, <Ref Constr="NewIdentityMatrix"/> returns a mutable identity
-##  matrix object with <A>m</A> rows and <A>n</A> columns
+##  matrix object with <A>m</A> rows and columns
```

Comment:
The declaration of `NewIdentityMatrix` introduces `n` in its `Arg` list, thus `<A>n</A>` should be used here.

Reply:
OK